### ISSUE TITLE
react-template-helper is still -rc0

### ISSUE THREAD

**Issue by acemtp** [meteor/react-packages#410]
Would be nice to have a 0.3.0 version without rc0 like all other packages when you released 3.0.

**Comment by denihs** [comment]
We're waiting on https://github.com/Meteor-Community-Packages/check-npm-versions/pull/46#issuecomment-2237201356.

After the official for `tmeasday:check-npm-versions` is out, we can release the official for this one.

**Comment by StorytellerCZ** [comment]
The PR was merged so we can proceed on this on now. @denihs

**Comment by denihs** [comment]
Hi, the official version has now been published: `react-template-helper@0.3.0`.
